pkg/network/ebpf/c/tracer/tracer.h

```diff
@@ -29,8 +29,16 @@ typedef enum
 typedef struct {
     __u64 sent_bytes;
     __u64 recv_bytes;
+    __u32 sent_packets;
```

Comment:
shouldn't these be lower?

Reply:
Doesn't make a difference if I move them further down, I still get the same 6 bytes of padding:
```
pkg/network/ebpf/c/tracer/tracer.h:29:9: warning: padding size of 'conn_stats_ts_t' with 6 bytes to alignment boundary [-Wpadded]
```

Reply:
The output is with the `-Wpadded` clang option btw.